### PR TITLE
Add mqtt_bridge

### PR DESCRIPTION
**Pull request by bartbart2003**

### CODE REVIEW

File: src/mqtt_bridge/msg/VescStatus.msg [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Header header
```
Comment (wisniax) [inline src/mqtt_bridge/msg/VescStatus.msg:1]: Message redefinition from can_wrapper should be avoided. Topic described [here](https://answers.ros.org/question/261641/how-to-use-custom-defined-messages-in-another-package/)

File: src/mqtt_bridge/msg/Wheels.msg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Header header
```
Comment (wisniax) [inline src/mqtt_bridge/msg/Wheels.msg:1]: Message redefinition from can_wrapper should be avoided. Topic described [here](https://answers.ros.org/question/261641/how-to-use-custom-defined-messages-in-another-package/)

File: src/mqtt_bridge/include/mqtt_bridge/ROSTopicHandler.hpp [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+private:
+  void publishMqttMessage(const std::string topicName, const char *message);
+  void addTimestampToJSON(rapidjson::Document &doc, long int nsec);
+  template<typename T>
+  void addMemberToJSON(rapidjson::Document &doc, std::string name, T value);
+  template<typename T>
+  void addMembersFromMapToJSON(rapidjson::Document &doc, const std::map<std::string, T>& m);
+
+  std::shared_ptr<mqtt::async_client> cli;
```
Comment (wisniax) [inline src/mqtt_bridge/include/mqtt_bridge/ROSTopicHandler.hpp:33]: I think there is some convention in cpp, that we decided to follow, to start class variables with `m` prefix, for e.g. mCli

File: src/mqtt_bridge/include/mqtt_bridge/ROSTopicHandler.hpp [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  int QOS;
+
+  const double interval_VescStatus = 0.05;
+  ros::Subscriber sub_VescStatus;
+  ros::Timer timer_VescStatus;
+  std::map<int, std::shared_ptr<mqtt_bridge::VescStatus>> msgMap_VescStatus;
+  void ROSTopicCallback_VescStatus(const mqtt_bridge::VescStatus::ConstPtr &receivedMsg);
```
Comment (wisniax) [inline src/mqtt_bridge/include/mqtt_bridge/ROSTopicHandler.hpp:40]: Why sudden change of naming convention going forward? Also not sure u need to add `ROSTopic` prefix to a function already in the `ROSTopic` class cuz invoking it looks kinda sus

**Review by wisniax — CHANGES_REQUESTED** [review]
Haven't looked at the cpp files yet, nonetheless:
## Good Job ;)

Just address the comments please.

**Review by AlvaroBajceps — APPROVED** [review]
Please consider sorting *src/mqtt_bridge/include/mqtt_bridge/ROSTopicHandler.hpp*

No other concerns from me.

**Review by wisniax — APPROVED** [review]

**Pull request merged**